### DISCUSSION TITLE
OKD 4.7 storage operator degraded - VSphereProblemDetectorController_SyncError

### DISCUSSION THREAD

**Discussion by schadom** [okd-project/okd#622] [Q&A] [answered]
**Describe the bug**
Cluster operator storage has been degraded for 10 minutes. Operator is degraded because VSphereProblemDetectorController_SyncError and cluster upgrades will be unstable.

VSphereProblemDetectorControllerDegraded: failed to connect to vcenter.corp.com: Post "https://vcenter.corp.com/sdk": dial tcp 192.168.144.100:443: i/o timeout

Cause:  Openshift Cluster in DMZ-segment with no access to vCenter API in Management-network.

**Version**
4.7.0-0.okd-2021-04-24-103438 (UPI on vSphere, without NSX(-T))

Is there any possiblity to disable VSphereProblemDetector checks or switch the cluster/provider from vSphere to bare-metal without having to reinstall the whole cluster?

**Comment by schadom** [comment]
Thanks @vrutkovs, will provide logs asap, although is more of a question than a bug. Is there any possiblity to disable VSphereProblemDetector checks or switch the cluster/provider from vSphere to bare-metal without having to reinstall the whole cluster?

**Comment by vrutkovs** [comment, marked as answer]
This check cannot be disabled - operator sets it and it cannot be excluded.
If you're not using vsphere features you should be using 'platform: none' to avoid this.

See bugzilla.redhat.com/show_bug.cgi?id=1943719 - it might be converted into an alert in the future

**Reply by schadom** [reply]
Thank you @vrutkovs . By chance, could you might guide me to some docs on how to update from platform:vsphere to platform:none?

**Reply by vrutkovs** [reply]
I don't think its possible now, you'd have to unfortunately reinstall the cluster
